rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -701,10 +703,10 @@ def get_balance(amount: str):
         return Balance(
             amount=FVal(amount), usd_value=FVal(amount) * CURRENT_PRICE_MOCK,
         )
-    assert blockchain.balances.eth[ethereum_accounts[0]].liabilities[A_USDC] == get_balance('166903.779933')  # noqa: E501
-    assert blockchain.balances.eth[ethereum_accounts[1]].assets[c_usdc_v3] == get_balance('0.32795')  # noqa: E501
-    assert blockchain.balances.eth[ethereum_accounts[2]].liabilities[A_USDC] == get_balance('257.565053')  # noqa: E501
-    assert blockchain.balances.eth[ethereum_accounts[3]].liabilities[A_USDC] == get_balance('589398.492789')  # noqa: E501
+    assert blockchain.balances.eth[ethereum_accounts[0]].liabilities[A_USDC] == get_balance('64394.356142')  # noqa: E501
+    assert blockchain.balances.eth[ethereum_accounts[1]].assets[c_usdc_v3] == get_balance('0.491925')  # noqa: E501
+    assert blockchain.balances.eth[ethereum_accounts[2]].liabilities[A_USDC] == get_balance('9378.084511')  # noqa: E501
+    assert blockchain.balances.eth[ethereum_accounts[3]].liabilities[A_USDC] == get_balance('0')
```

Comment:
you shouldn't need to edit other tests more if they are not related. This test compounds, not sure why gearbox affects to it

Reply:
Fixed by patching `ensure_cache_data_is_updated`